solr/solrj/src/java/org/apache/solr/client/solrj/impl/Http2SolrClient.java

```diff
@@ -115,6 +115,11 @@ public class Http2SolrClient extends HttpSolrClientBase {
   private SSLConfig sslConfig;
 
   private List<HttpListenerFactory> listenerFactory = new ArrayList<>();
+
+  public List<HttpListenerFactory> getListenerFactory() {
```

Comment:
Minor, but this getter is very misplaced.  It shouldn't be placed amidst the fields.